aws/service.go

```diff
@@ -1683,6 +1684,30 @@ func WAFService(ctx context.Context, d *plugin.QueryData) (*waf.WAF, error) {
 	return svc, nil
 }
 
+// WAFRegionalService returns the service connection for AWS WAF Regional service
+func WAFRegionalService(ctx context.Context, d *plugin.QueryData) (*wafregional.WAFRegional, error) {
+	// have we already created and cached the service?
```

Comment:
Is this commented line required?

Reply:
Yes, below line of code return the cached data if any. we have this comment for all the functions.